### DISCUSSION TITLE
Galaxy Brain Achievement

### DISCUSSION THREAD

**Discussion by ravi326** [Schweinepriester/github-profile-achievements#369] [Q&A] [answered]
Does the achievement mean:
"You need to open 2 discussions that are answered" or "You need to become the marked answer to 2 discussions?"

**Comment by scarabaeus** [comment]
"You need to become the marked answer to 2 discussions?"

**Reply by tanish36** [reply]
help

**Comment by lucifer991** [comment, marked as answer]
You need to become the marked answer to 2 discussions!

**Comment by longquoc47krb** [comment]
You need to become the marked answer to 2 discussions?
